### PR TITLE
Completed html-web-page Coding Challenge - Bingle Padilla 

### PR DESCRIPTION
**Pull request by codewithbingle**
html-web-page

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        </tr>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+    <p><h4>Like what you see? Become a part of the JOTMC family today!</h4>
```
Comment (Zhiwutian) [inline html-web-page/index.html:72]: you can remove this extra `p` opening tag

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update! Looks great!

File: html-web-page/index.html (hunk 1 above)
Comment (codewithbingle) [reply html-web-page/index.html]: Updated.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**